bool ifVerbose
---
geometry_msgs/Pose eef_pose
